src/board.js

```diff
@@ -1,23 +1,33 @@
-const print = console.log;
+function createBoard(size) {
+    if (size <= 0) {
+        throw new Error(`The board size must be positive: got [${size}].`)
+    }
+    const board = new Array(size);
+    board.forEach((cell, index) => {
+        board[index] = new Array(size);
+    });
+    return board;
+}
```

Comment:
Here's the `createBoard` function that you coded; it has:

* some defensive checking at the start because it doesn't make sense to create a board with negative spaces.
* some initialization code to ensure that the board is square.